src/plugins/mer/merrunconfigurationaspect.cpp

```diff
@@ -242,9 +255,20 @@ void MerRunConfigurationAspect::applyTo(ProjectExplorer::Runnable *r) const
                                    QLatin1String(QMLLIVE_SAILFISH_PRELOAD),
                                    QLatin1String(LIST_SEP));
 
-        if (qmlLiveIpcPort() > 0) {
+        int qmlLiveIpcPort = this->qmlLiveIpcPort();
```

Comment:
Maybe use `port`, `adjustedPort` or `adjustedQmlLiveIpcPort` instead of a method shadowing?

Reply:
I thought hardly what could be a more appropriate name.. I think we should rather keep using the same name.